app/details/[ticket_id]/ticketDetails.tsx

```diff
@@ -88,6 +88,18 @@ function TicketDetails(props: Props) {
     setModeSelectedItem(item);
   };
 
+  const handleMacroSelect = useCallback(
+    (selectedMacro: { content: string }) => {
+      setCommentValue((prevValue) => {
```

Comment:
Use here this instead of append content
setCommentValue(selectedMacro.content);